js/app/improve-this-page.js

```diff
@@ -36,55 +36,6 @@ $(document).ready(function () {
         }
     });
 
-    $("#feedback-form-yes").click(function (e) {
-        e.preventDefault();
-        var postData = $("#feedback-form-container").serialize();
-        var postObject = new Object();
-        postObject.is_page_useful = true;
-        postObject.is_general_feedback = false;
-        postObject.ons_url  = window.location.href;
-        var postJson = JSON.stringify(postObject);
-
-        if (useFeedbackAPI && useFeedbackAPI.value === "true") { 
-            $.ajax({
-                type: "POST",
-                url: feedbackURL,
-                dataType: 'json',
-                processData: false ,
-                data: postJson,
-                contentType: "application/json",
-                beforeSend: function () {
-                    var formHeader = $("#feedback-form-header")
-                    $("#feedback-form").addClass("js-hidden");
-                    formHeader.removeClass("js-hidden");
-                },
-                success: function () {
-                    $("#feedback-form-header").html(feedbackMessage);
-                },
-                error: function () {
-                    $("#feedback-form-header").html(feedbackErrorMessage);
-                },
-            })
-        } else {
-            $.ajax({
-                type: "POST",
-                url: feedbackURL,
-                data: postData,
-                beforeSend: function () {
-                    var formHeader = $("#feedback-form-header")
-                    $("#feedback-form").addClass("js-hidden");
-                    formHeader.removeClass("js-hidden");
-                },
-                success: function () {
-                    $("#feedback-form-header").html(feedbackMessage);
-                },
-                error: function () {
-                    $("#feedback-form-header").html(feedbackErrorMessage);
-                },
-            })            
-        }
-    });
-
```

Comment:
This looks to be removing code when you click the 'yes' button in the footer - what happens instead?

Reply:
You are correct, removing this redirects the user to `/feedback/thanks` 
Adding it again, with slight changes to how we post to the `feedback-api` and a description message instead of blank